ckanext/datajson/harvester_base.py

```diff
@@ -430,6 +431,16 @@ def import_stage(self, harvest_object):
                     'Import')
                 return None
 
+
+        # do title check here
```

Comment:
Having a hard time to capture the ValidationError on line #714. Doing a `_save_object_error` and `return None` there always gives some sqlalchemy session error. So I import the function from core to this line and do extra check, where I can execute `_save_object_error`